src/approaches/oracle_approach.py

```diff
@@ -208,3 +210,106 @@ def grasp_sampler(state: State, rng: np.random.Generator,
     operators.add(dump_operator)
 
     return operators
+
+
+def _get_blocks_gt_ops() -> Set[Operator]:
+    """Create ground truth operators for BlocksEnv.
+    """
+    block_type, robot_type = _get_types_by_names("blocks", ["block", "robot"])
+
+    On, OnTable, GripperOpen, Holding, Clear = _get_predicates_by_names(
+        "blocks", ["On", "OnTable", "GripperOpen", "Holding", "Clear"])
+
+    Pick, Stack, PutOnTable = _get_options_by_names(
+        "blocks", ["Pick", "Stack", "PutOnTable"])
+
+    operators = set()
+
+    # PickFromTable
+    block = Variable("?block", block_type)
+    robot = Variable("?robot", robot_type)
+    parameters = [block, robot]
+    option_vars = [block]
+    option = Pick
+    preconditions = {LiftedAtom(OnTable, [block]),
+                     LiftedAtom(Clear, [block]),
+                     LiftedAtom(GripperOpen, [robot])}
+    add_effects = {LiftedAtom(Holding, [block])}
+    delete_effects = {LiftedAtom(OnTable, [block]),
+                      LiftedAtom(Clear, [block]),
+                      LiftedAtom(GripperOpen, [robot])}
+    def pick_sampler(state: State, rng: np.random.Generator,
+                     objs: Sequence[Object]) -> Array:
+        del state, rng, objs  # unused
+        return np.zeros(3, dtype=np.float32)
```

Comment:
what's the point of sampling a non-zero-shape vector if it's deterministic? can't we just have the pick option always output the same action, like Dump? same question for the other options (except put on table)

Reply:
these are oracle samplers. we would want our system to learn this (admittedly easy) sampler. i know it's a trivial thing to learn but it's just for sanity checking our sampler learning